q2_quality_control/plugin_setup.py

```diff
@@ -65,7 +69,10 @@
         'filter, e.g., extract only feature sequences that align to a certain '
         'clade of bacteria; or to define a negative filter, e.g., identify '
         'sequences that align to contaminant or human DNA sequences that '
-        'should be excluded from subsequent analyses.')
+        'should be excluded from subsequent analyses. Note that filtering is '
```

Comment:
You'll need to add `perc_query_aligned` here as well (just a reminder).

Reply:
👍 thanks!